optional slot mappings (#7171)
RasaHQ/rasa: docs/docs/migration-guide.mdx

            return {"cuisine": None}
```

> It is also possible to migrate forms to Rasa Open Source iteratively. You can for

Reviewer: ```suggestion
You can also migrate forms from Rasa SDK to Rasa Open Source 2 iteratively. You can for
```

example migrate one form to the Rasa Open Source 2 implementation while continue using

> the deprecated Rasa SDK implementation for another form. If you want to continue to use
> the deprecated Rasa SDK `FormAction`s, then you can instruct Rasa Open Source to do so
> by adding a custom action with the name of your form to your domain. Note that it is
> recommended to complete the migration as soon as possible as the deprecated `FormAction`

Reviewer: ```suggestion
the deprecated Rasa SDK implementation for another form. To continue to use
the deprecated Rasa SDK `FormAction`s, add a custom action with the name of your form to your domain. Note that you should complete the migration as soon as possible as the deprecated `FormAction`
```
